owncloudApp/src/main/res/values/strings.xml

```diff
@@ -43,6 +43,15 @@
     <string name="prefs_passcode">Passcode lock</string>
     <string name="prefs_biometric">Biometric lock</string>
     <string name="prefs_biometric_summary">Enable passcode or pattern lock to enable this option</string>
+    <string name="prefs_lock_application">Lock application</string>
+    <string name="prefs_lock_application_entries_immediately">Immediately</string>
+    <string name="prefs_lock_application_entries_1minute">After 1 minute</string>
+    <string name="prefs_lock_application_entries_5minutes">After 5 minutes</string>
+    <string name="prefs_lock_application_entries_30minutes">After 30 minutes</string>
+    <string name="prefs_access_from_document_provider">Access from document provider</string>
+    <string name="prefs_access_from_document_provider_summary">Allow access to the files of ownCloud accounts in the app via the Android native file explorer when there is a lock set.</string>
```

Comment:
`ownCloud` word should be replaced by the proper parameter appname

Reply:
I rewrote it avoiding the word ownCloud